billy-core/src/main/java/com/premiumminds/billy/core/services/builders/impl/GenericInvoiceEntryBuilderImpl.java

```diff
@@ -80,7 +87,7 @@ public GenericInvoiceEntryBuilderImpl(TDAOEntry daoEntry, TDAOInvoice daoInvoice
     @NotOnUpdate
     public <T extends ShippingPoint> TBuilder setShippingOrigin(Builder<T> originBuilder) {
         BillyValidator.notNull(originBuilder,
-                GenericInvoiceEntryBuilderImpl.LOCALIZER.getString("field.entry_shipping_origin"));
+                               GenericInvoiceEntryBuilderImpl.LOCALIZER.getString("field.entry_shipping_origin"));
```

Comment:
The way it is presented might be misleading but it doesn't include tabs, it's all indented with spaces. I ran the spotless plugin just in case and resulted in no changes on the project.